packages/taquito/src/subscribe/create-observable-from-subscription.ts

```diff
@@ -0,0 +1,25 @@
+import { BlockResponse } from '@taquito/rpc';
+import { Observable } from 'rxjs';
+import { OperationContent, Subscription } from './interface';
+
+export function createObservableFromSubscription<
```

Comment:
can you give a brief explanation of what an 'observable' is in this case?

Reply:
The observable will emit every time there is a new block. In the operation confirmation methods, we subscribe to the observable until the hash of the operation is seen in one of the blocks received (or until the correct number of confirmation is reached which is the number of blocks received after the operation hash was found).